modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of the AWS Load Balancer Controller using ALB Operator
+
+After installing the operator, you can create an instance of the `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator reconciles only the resource with the name `cluster`.
+
+.Prerequisites
+
+. You have created the `echoserver` namespace.
+
+.Procedure
+
+. Create a `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: Auto <3>
+  additionalResourceTags: <4>
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud <5>
+  config:
+    replicas: 2 <6>
+  enabledAddons: <7>
+    - AWSWAFv2 <8>
+----
+
+<1> Defines the `AWSLoadBalancerControlller` resource.
+<2> Defines the `aws-load-balancer-controller` instance name. This instance name gets added as a suffix to all related resources.
+<3> It can define two options such as `Auto` and `Manual`. When the value is set to `Auto` the operator attempts to determine the subnets which belong to the cluster and tags them appropriately. The operator cannot determine the role correctly if the internal subnet tags are not present on internal subnet. So if you installed your cluster on User-Provisioned Infrastructure, you must manually tag the subnets with the appropriate role tags and set the subnet tagging policy to `Manual`.
```

Comment:
Instead of "It can define two options such as `Auto` and `Manual" I would just say "Valid options are `Auto` and `Manual`. When the value is set to `Auto`, the Operator attempts to determine the subsets that belong to the cluster, then tags them appropriately.

Reply:
If you installed your cluster on the user-provided intrastructure,

Reply:
Ack'ed